composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
I'm thinking more, should we do this if we are doing this (because it will spread into the other 3.2.x extensions then too)? Or drop it for now, until you guys have sorted out your think about this?